### ISSUE TITLE
NEUROARCH-008: Preregistered architecture tournament and qualification receipt

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#3324]
Parent: #3316
Depends on: qualified #3317, complete #3318 stack (#3552–#3555), and qualified/characterized #3319–#3323

## Goal
Run the first **preregistered held-out neuroarchitecture evaluation** under matched-resource profiles and emit auditable evidence receipts.

The tournament does not need to produce one winner. Its purpose is to determine whether any preregistered candidate clears all required capability, robustness, calibration, determinism and resource gates relative to the qualified incumbent/control set.

Multiple candidates may qualify. No candidate may qualify. `Inconclusive` and `NoCandidateQualified` are valid scientific outcomes.

## Evidence-before-interpretation rule

Candidate interpretation is permitted only after #3555's conformance oracle emits a `QualifiedEvidenceSet` for the tournament lineage.

```text
raw trial receipts
      ↓
#3555 conformance / held-out governance
      ↓
QualifiedEvidenceSet ?
   no → no candidate conclusion
   yes
      ↓
preregistered tournament analysis
      ↓
candidate qualification statuses
```

A contaminated/malformed/incomplete held-out evidence set cannot be rescued by attractive task results.

## Preregistration freeze

Before held-out reveal/execution, commit a canonical preregistration package binding at least:
- tournament schema/profile version;
- exact code/toolchain/lock/build lineage;
- evidence-stack schema/profile versions;
- entrant set and admissibility rules;
- fixed-topology vs topology-family interpretation for each entrant;
- exact topology/generator/subject/checkpoint commitments as applicable;
- topology-generation seeds where applicable;
- subject-initialization seeds;
- task/trial seeds;
- Dev/Validation/HeldOut partition commitments;
- task-family/fixture commitments;
- exact ordered dt/time schedules where temporal;
- primary endpoints;
- secondary/exploratory endpoints;
- regression-veto endpoints;
- failure/retry/exclusion rules;
- matched-resource profile(s) and tolerances;
- effect-size/non-inferiority/superiority margins;
- statistical model/aggregation/multiplicity policy;
- minimum valid run/topology/seed requirements;
- stopping rule;
- candidate qualification logic;
- evidence-set conformance profile;
- independent-verifier version/profile.

After tuning closes, changing any of these creates a new preregistration/evidence lineage.

## Entrant semantics

Distinguish two kinds of claims:

### Fixed topology
A specific realized `TopologyCommitment` frozen after development is evaluated across preregistered subject/task seeds.

A positive result applies to that realized architecture under the tested subject/config family; it does not automatically establish that the topology generator generalizes.

### Topology family
A deterministic generator/config family is evaluated across preregistered **topology-generation seeds** distinct from subject/task seeds.

Family-level analysis treats topology realization as a sampling level rather than pseudoreplicating all task runs as independent graphs.

A tournament receipt must say which interpretation applies to each entrant.

## Minimum entrant set

Freeze exact identities before held-out evaluation. At minimum retain representative anchors equivalent to:

1. qualified active layered HDC/LTC incumbent;
2. matched random/recurrent control;
3. modular-only control;
4. timescale/hierarchical-only control;
5. modular + rich-club control;
6. final preregistered combined engineered HLC candidate from #3321;
7. task-blind FlyWire-statistical control from #3323.

Additional factorial controls may remain if preregistered, but entrant count cannot expand after held-out results are visible.

The FlyWire control is a biological prior, not a privileged benchmark winner or architecture template.

## Three independent seed axes

Never collapse these into a generic seed:

```text
TopologyGenerationSeed
SubjectInitializationSeed
TaskTrialSeed
```

When an entrant is a fixed topology, the first axis is fixed/not-applicable but remains conceptually distinct.

Use paired subject/task seeds across compatible entrants wherever scientifically appropriate to reduce variance. The pairing structure is receipt-bound.

## Avoid pseudoreplication

The statistical unit must match the claim.

For topology-family claims, repeated task trials on one realized topology do not count as independent topology samples.

Preregister a hierarchical/cluster-aware analysis or another defensible aggregation structure that preserves:
- topology-realization variation;
- subject-initialization variation;
- task/trial variation.

Report all levels rather than only pooled means.

## Opaque entrant evaluation

Where practical, metric computation and qualification logic operate on deterministic opaque entrant IDs. Human-readable architecture names are mapped back only after raw result/analysis commitments are frozen.

This is not claimed to be perfect blinding, but it reduces opportunities for architecture-specific manual exceptions during scoring.

## Primary question structure

Separate factor questions from final-candidate qualification.

Examples of preregistered scientific questions:
- modularity effect under matched resources;
- rich-club benefit beyond degree/fan-out/message budget;
- multi-timescale benefit beyond the same timescale multiset;
- factor interactions in the combined HLC;
- FlyWire-prior utility versus appropriate degree/statistical controls;
- perturbation/recovery robustness;
- capability-resource Pareto tradeoffs.

The tournament must not collapse all questions into one overall architecture score.

## Task compatibility

Primary matched comparisons use task profiles for which every required entrant is compatible under the same semantic task definition.

If an entrant lacks a required capability:
- handle it under a preregistered incompatibility/failure rule;
- do not silently change the task for that entrant;
- do not grant architecture-specific preprocessing or timing semantics off-book.

## Qualification gates are conjunctive

Candidate research qualification is **not** a mean-score competition.

Define independent preregistered gates, for example:

### G0 Evidence conformance
- tournament evidence set is `QualifiedEvidenceSet` under #3555;
- exact preregistration lineage intact;
- no held-out contamination.

### G1 Determinism / reproducibility
- deterministic profiles reproduce required trace commitments within the exact profile's rule;
- no unexplained semantic replay divergence;
- golden/conformance receipts valid.

### G2 Validity / failure rate
- candidate-caused crash, non-finite, invalid-trial and resource-violation rates stay within preregistered limits/margins;
- missing/failed runs are not silently removed.

### G3 Primary capability
- preregistered primary task endpoints meet their superiority or non-inferiority margins under the frozen statistical procedure.

### G4 Regression vetoes
Candidate must not materially regress preregistered critical dimensions such as:
- calibration;
- temporal/continual stability;
- interference;
- robustness/recovery;
- determinism;
- failure rate;
- other safety/reliability measures frozen before reveal.

### G5 Resource envelope
- strict matched-resource comparison passes the named #3553 profile/tolerances;
- a candidate outside the envelope may still be reported in unconstrained/Pareto analysis but cannot pass the strict matched qualification gate.

### G6 Perturbational robustness
- preregistered #3322 stress/effectome-derived robustness criteria are met where required;
- intervention-induced failures remain evidence.

A candidate qualifies only if **all mandatory gates** pass. A strong result on one dimension cannot compensate for a vetoed dimension through an aggregate score.

## Effect size and uncertainty

Do not qualify based on point estimates alone.

Preregister:
- effect-size definition;
- uncertainty interval/profile;
- superiority or non-inferiority margin;
- paired/hierarchical analysis structure;
- multiplicity policy across primary endpoints/comparisons;
- minimum sample/topology counts.

`p < threshold` alone is not a sufficient architecture qualification rule, and `not significant` is not automatically evidence of equivalence.

## Multiplicity

The tournament tests multiple architectures/endpoints. Freeze the confirmatory comparison family and correction/hierarchical procedure before held-out reveal.

Exploratory analyses may be extensive but remain explicitly exploratory and cannot retroactively qualify an entrant.

## Resource analyses

Maintain at least two separate views:

### Strict matched qualification
Uses a frozen #3553 composite match profile and required tolerances.

### Unconstrained/Pareto characterization
Shows capability/resource tradeoffs without pretending entrants are matched.

Do not mix these evidence types in one promotion gate.

Physical latency/energy results remain bound to environment/method identity. Semantic task/trace evidence is not defined by scheduler noise.

## Effectome/robustness integration

Use only preregistered #3322 intervention profiles/observables/horizons for confirmatory robustness gates.

Effectome exploratory scans performed after held-out reveal cannot become new qualification endpoints without a new lineage.

## Failure and retry discipline

Use #3555 taxonomy/retry policy.

Key rules:
- valid poor outcomes are not retryable merely because they are poor;
- candidate crashes/non-finites/resource violations remain candidate failures;
- harness failures are separately invalidated;
- infrastructure-indeterminate runs may be retried only under the frozen retry policy;
- all attempts remain evidence-linked;
- selective successful-attempt reporting is forbidden.

## No result peeking / early stopping

Do not stop when a preferred architecture looks good or bad unless a sequential stopping design was preregistered.

Held-out raw results should remain machine-processed under the frozen analysis path until the primary receipt is committed.

Any human-directed post-hoc exploration starts a clearly labeled exploratory branch after the primary receipt exists.

## Independent verification

The final tournament needs an independent verifier that recomputes, from raw canonical receipts:
- entrant/trial inclusion;
- partition/conformance state;
- failure census;
- matched-resource assessments;
- aggregate/effect estimates;
- qualification gates;
- final candidate statuses.

Prefer a separately implemented oracle/code path rather than calling the same functions twice.

The primary and independent verifier outputs must agree under a canonical comparison rule before candidate qualification can be asserted.

## Replication level

At minimum distinguish:
- exact deterministic replay replication;
- repeated subject/task-seed replication;
- topology-family replication where claimed;
- environment-bound physical-performance replication.

A semantic qualification receipt must not overclaim physical-performance portability from one machine/environment.

## Candidate status vocabulary

Keep evidence-set and candidate statuses distinct.

### Evidence-set status (#3555)
```text
QualifiedEvidenceSet
NotQualified
Inconclusive
InfrastructureIndeterminate
```

### Candidate tournament status
Use an equivalent explicit vocabulary such as:

```text
ResearchQualified
NotQualified
Inconclusive
NotComparable
```

`ResearchQualified` means only that the candidate cleared this preregistered experimental gate. It is **not** a production authorization or consciousness claim.

Multiple candidates may be `ResearchQualified`.

## `NeuroarchTournamentReceipt`

Bind at minimum:
- receipt/preregistration schema/profile;
- preregistration commitment;
- #3555 qualified evidence-set receipt root;
- exact code/toolchain/lock/build lineage;
- entrant identity mapping and opaque IDs;
- topology/generator/subject/checkpoint roots;
- partition and seed roots;
- raw per-trial receipt root;
- failure/retry/exclusion census;
- raw metric dataset root;
- matched-resource assessment roots;
- topology/effectome/robustness evidence roots;
- statistical-analysis profile/result roots;
- independent-verifier root;
- every mandatory gate result per candidate;
- candidate statuses;
- exploratory-analysis roots, clearly separated;
- interpretation/evidence-boundary statement.

The receipt should make it possible to audit *why* a candidate did or did not qualify without rerunning the experiment.

## Negative/null evidence

Publish/retain:
- no-factor-effect results;
- failed candidate gates;
- resource tradeoffs;
- architecture-specific failure modes;
- FlyWire prior null/negative results;
- `NoCandidateQualified` outcome.

No gate is weakened because the result is disappointing.

## Acceptance

- held-out preregistration committed before reveal;
- evidence set independently qualifies under #3555 before candidate interpretation;
- exact entrant set/identities frozen;
- seed axes and pairing frozen;
- fixed-topology vs topology-family claim type explicit;
- analysis avoids topology-level pseudoreplication;
- opaque-ID metric/qualification path used where practical;
- all mandatory gates are conjunctive and independently reported;
- resource matching uses explicit #3553 profile and exposes residuals;
- primary statistical/multiplicity procedure frozen;
- failure/retry/exclusion census complete;
- deterministic replay and independent verifier agree;
- negative/null/inconclusive results retained;
- more than one or zero candidate may qualify;
- no scalar leaderboard/winner is required;
- no production default switch occurs in this PR;
- no consciousness or biological-equivalence claim follows from qualification.

## Post-tournament boundary

A `ResearchQualified` result authorizes only a **separate replication/integration review**. Production-default changes require a new explicitly scoped gate with independent replication, compatibility/safety testing, rollback criteria and reviewed migration evidence.

## Non-goals
- no same-PR production switch;
- no automatic selection of one winner;
- no relaxation of gates after held-out reveal;
- no consciousness inference;
- no claim that FlyWire similarity implies superiority.

**Comment by Tristan-Stoltz-ERC** [comment]
Tournament anti-overfitting rule: architecture selection/hyperparameter tuning ends before held-out commitments are revealed. Development runs may eliminate obviously bad candidates but may not redefine the primary metric or qualification threshold after observing held-out results. Any post-hoc analysis must be labeled exploratory.

**Comment by Tristan-Stoltz-ERC** [comment]
Qualification robustness rule: a candidate with better mean task performance but materially worse perturbation recovery, calibration, determinism, or normalized resource use must not be silently promoted. Regression vetoes are first-class preregistered criteria.

**Comment by Tristan-Stoltz-ERC** [comment]
Final-report rule: include negative/null results and architecture-specific failure modes in the qualification receipt. If no entrant qualifies, `Inconclusive`/`NotQualified` is a successful scientific outcome, not a reason to weaken gates.
